contracts/LuniverseGluwacoinGateway.sol

```diff
@@ -55,22 +55,26 @@ contract LuniverseGluwacoinGateway is Initializable, ContextUpgradeSafe, AccessC
     }
 
     /**
-     * @dev Returns the address of the base token.
+     * @notice Returns the address of the base token.
      */
     function token() external view returns (IERC20) {
         return _token;
     }
 
     /**
-     * @dev Returns if there is Unpeg for the {txnHash}.
+     * @notice Returns if there is a Unpeg record for the {txnHash}.
      */
     function isUnpegged(bytes32 txnHash) external view returns (bool unpegged) {
         return _isUnpegged(txnHash);
     }
-
-    /**
-     * @dev Returns Unpeg for the {txnHash}.
-     */
+  
+    /// @notice return an unpeg record
+    /// @param txnHash a bytes32 string to be used as an identifier for each unpeg record  
+    /// @return amount: to be unpegged
+    /// @return sender: the address to receive the unpgged token less fee (if applicable)
+    /// @return gluwaApproved: indicate whether the unpeg record is approved by Gluwa
+    /// @return gluwaApproved: indicate whether the unpeg record is approved by Luniverse
+    /// @return processed: indicate whether the unpeg record is processed    
```

Comment:
/// @return amount : to be unpegged
    /// @return sender : the address to receive the unpgged token less fee (if applicable)
    /// @return gluwaApproved : indicate whether the unpeg record is approved by Gluwa
    /// @return luniverseApproved : indicate whether the unpeg record is approved by Luniverse
    /// @return processed : indicate whether the unpeg record is processed    

need a space between param name and colon
line 76 gluwaApproved => luniverseApproved